OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -176,6 +189,18 @@ public void Repulse()
 			SetPosition(self, CenterPosition + FlyStep(speed, repulsionForce.Yaw.Facing));
 		}
 
+		public void DisableRepulsing()
```

Comment:
This looks dangerous.  I would strongly prefer it if we didn't do this – we have wasted so much time over the years debugging issues with manually managed state like this.